src/components/htmlMediaHelper.js

```diff
@@ -131,8 +131,11 @@ import { Events } from 'jellyfin-apiclient';
     }
 
     function setCurrentTimeIfNeeded(element, seconds) {
-        if (Math.abs(element.currentTime || 0, seconds) <= 1) {
-            element.currentTime = seconds;
+        // If it's worth skipping (1 sec or less of a difference)
+        if (Math.abs((element.currentTime || 0) - seconds) >= 1) {
+            // Changing currentTime directly doesn't seek in the video
+            // A <=500 timeout doesn't seem to work for my setup
+            setTimeout(() => { element.currentTime = seconds; }, 500);
```

Comment:
Probably worth more investigation in the future...
I also tried using the "Using mediaOption to Avoid Needless Preload" [described here](http://webcache.googleusercontent.com/search?q=cache:DCB3EzosswEJ:https://webostv.developer.lge.com/develop/developer-guide/resuming-media-quickly-mediaoption/&hl=en&gl=il&strip=1&vwsrc=0) but to no avail.

Great suggestions! Pushed